src/components/NoticeBox/WarningBox.js

```diff
@@ -0,0 +1,14 @@
+import { NoticeBox } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+import React from 'react'
+
+export const WarningBox = ({ title, text }) => (
```

Comment:
Hey,
IMHO there is a mismatch between the name `WarningBox` and the `error` version of the NoticeBox component. I suggest either renaming this component to `ErrorBox` or using [warning](https://ui.dhis2.nu/components/notice-box/#warning) instead. WDYT?
Thanks!